content/docs/start/data-versioning.md

```diff
@@ -1,3 +1,8 @@
+---
+description: |
+  Get started with data versioning in DVC. Learn how to use a regular Git workflow for large files, datasets, and ML models, without storing large files in Git.
```

Comment:
Hmmm. Restyled doesn't pick this up but it should be wrapped at 80 chars like in blogs, I think.

Reply:
Removing a term that repeats:
```suggestion
  Get started with data versioning in DVC. Learn how to use a regular Git workflow for datasets, and ML models, without storing large files in Git.
```

Reply:
- I see this comes from the GS index, good idea. Should we also include "Git for data" somewhere in the docs?
- "ML models" is also good but should the full term "machine learning" or "machine learning models" (and possibly other terms around "model" like "model versioning" be added somewhere to the doc too? It's not present at the moment.

Reply:
> Hmmm. Restyled doesn't pick this up but it should be wrapped at 80 chars like in blogs, I think.

Yeah, fixed it now. FYI when I used the `|` character and tried to format it as in blog posts it broke the renderer. Seems work fine with single quotes like in Roger's initial example.

> Removing a term that repeats

Nice catch, thanks. Fixed and also removed extra comma.

> Should we also include "Git for data" somewhere in the docs?

Good idea. Added in rewrite of intro section that I just pushed. PTAL

> "ML models" is also good but should the full term "machine learning" or "machine learning models" (and possibly other terms around "model" like "model versioning" be added somewhere to the doc too? It's not present at the moment.

In this case I was only looking to add "data versioning". As per your request for one term/doc/PR, but also the subject matter and existing terms for this page are more related to versioning than ML. So it's OK to keep with the abbreviations here and apply the full set in other places.

Reply:
Sounds good.